bgpd/bgp_bmp.h

```diff
@@ -177,6 +177,7 @@ struct bmp_active {
 	struct bmp *bmp;
 
 	char *hostname;
+	union sockunion *srcaddr;
```

Comment:
i used sockunion since that type is already use in that bmp_active struct.
that union support ipv6 and ipv4 address.

in term of performance one allocation at start is not an issue
but if you prefer to have union sockunion  rather union sockunion * and thus avoid a allocation and a free, i can do it let me know

Reply:
union sockunion *  changed to union sockunion